web/src/components/itemList.js

```diff
@@ -67,11 +71,12 @@ class ItemList extends Component {
                         <br />
                         <Button variant="link" style={{padding: '0'}} id={i} onClick={this.currencyConvert}>{this.state.currency[i] ? 'Convert To KRW' : 'Convert To USD'}</Button>
                     </Card.Text>
-                    <Button variant={
-                        this.selected[i] ? 'secondary' : 'primary'
-                    } id={i} onClick={this.itemClicked}>{
-                        this.selected[i] ? 'Remove from cart' : 'Add to cart'
-                    }</Button>
+                    
+                    <Button id={i} variant='primary' onClick={this.itemAdd}>Add this stuff</Button>
+                    <Card.Text>
+                        <NumberFormat value={this.selected[i]} displayType={'text'} thousandSeparator={true} prefix="담은 개수: " suffix="개" />
```

Comment:
If you want to update the shape of the component, you should use state other than a general object.